renderer-macos/lib/components/SegmentedControl.re

```diff
@@ -0,0 +1,117 @@
+module OutputTree = {
+  type node =
+    | SegmentedControl(CocoaTypes.view)
+    | Item(BriskSegmentedControl.Item.t);
+
+  let markAsStale = () => ();
+
+  let insertNode = (~parent: node, ~child: node, ~position: int) => {
+    switch (parent, child) {
+    | (SegmentedControl(parent), Item(child)) =>
+      BriskSegmentedControl.insertItem(parent, child, position)
+    | _ => assert(false)
+    };
+    parent;
+  };
+
+  let deleteNode = (~parent, ~child) => {
```

Comment:
This will be much better when https://github.com/briskml/brisk-reconciler/issues/10 is solved.